### DISCUSSION TITLE
用 docker-compose 装的，clickhouse连不上

### DISCUSSION THREAD

**Discussion by JellyZero** [clickvisual/clickvisual#876] [Q&A] [answered]
环境: mac m1

<img width="973" alt="image" src="https://user-images.githubusercontent.com/109566229/211449826-04020fab-07ec-4277-bf01-fc2ee5b2632a.png">

<img width="503" alt="image" src="https://user-images.githubusercontent.com/109566229/211449853-8e9307f3-b437-45eb-adf9-830470bf0aa2.png">

**Comment by JellyZero** [comment]
<img width="1740" alt="image" src="https://user-images.githubusercontent.com/109566229/211449901-d56065e0-2611-440e-8755-86aa09d4652e.png">

**Reply by kl7sn** [reply]
使用 clickhouse:9000 试试，这边之后再自测下 docker-compose 流程。

**Comment by kl7sn** [comment]
clickhouse://root:shimo@clickhouse:9000/default?max_execution_time=60

**Reply by JellyZero** [reply, marked as answer]
不加 tcp:// 连不上

**Comment by kl7sn** [comment]
环境：Apple M1
我这边运行的是 master 上的 docker-compose.yml 

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/2037801/211464662-e38bc5a5-4d7a-4f3b-aed1-d5b96e55c132.png">
<img width="344" alt="image" src="https://user-images.githubusercontent.com/2037801/211464701-06e0c209-1378-402c-99f7-ded678db8bea.png">
